assets/app/scripts/controllers/overview.js

```diff
@@ -8,7 +8,7 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('OverviewController', function ($scope, DataService, $filter, LabelFilter, Logger, ImageStreamResolver) {
+  .controller('OverviewController', function ($scope, DataService, $filter, LabelFilter, Logger, ImageStreamResolver, ObjectDescriber, $rootScope) {
```

Comment:
we fixed the minify issue by locking to a previous version, but yes they are unused now so i will remove them